code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -2154,6 +2178,22 @@
 	glass_desc = "A glass of refreshing Comet Cola"
 	glass_center_of_mass = list("x"=17, "y"=6)
 
+/singleton/reagent/drink/coca_cola
+	name = "Coca Soda"
```

Comment:
```suggestion
	name = "Coca Cola"
```
might aswell call it that, since both coca and cola are generic names. note that the corporation uses a hyphen.